### PR TITLE
Countries, cities and states appearing as drop down

### LINKED ISSUE #454: Countries should appear as dropdowns in volunteer search and sign up forms
Right now the users have to manually type the country in the fields. It'd be great if users can select country through a drop down list. In that case, even validation won't be required. Also, this is implemented already in systers/portal which vms is going to be a part of (correct me if i'm wrong :smile:)

more of an enhancement, IMO.

### PR DESCRIPTION
**Pull request by anjali-dhanuka**
### Description
Django cities light package implemented thus countries, cities and states appearing as drop down.

Fixes #454 

### Type of Change:
- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

How Has This Been Tested?
screenshot from 2018-01-16 22-55-33
image
image

image
screenshot from 2018-01-16 22-55-14

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by anjali-dhanuka** [comment]
@tapasweni-pathak @tonythomas01 Please review.

**Comment by tonythomas01** [comment]
@anjali-dhanuka look at how I implemented it for the event dropdowns at https://github.com/systers/vms/pull/511

**Comment by anjali-dhanuka** [comment]
@tonythomas01 thanks a lot for the guidance. I have implemented it. Please have a look.

**Comment by anjali-dhanuka** [comment]
@tapasweni-pathak @tonythomas01 
Build log is now showing. Please review.

**Comment by anjali-dhanuka** [comment]
@tonythomas01 @tapasweni-pathak we need to make some changes in .travis file for populating the database. Please guide me how do I proceed.

**Comment by anjali-dhanuka** [comment]
@tonythomas @tapasweni-pathak Need  help  with travis.

File: vms/volunteer/services.py [hunk 1]
```diff
@@ -6,7 +6,12 @@
 from organization.services import (get_organization_by_name,
                                    get_organizations_ordered_by_name)
 from volunteer.models import Volunteer
+from volunteer.models import City
 
+def get_cities():
+    
+    city_list = City.objects.all()    
+    print city_list
```
Comment (tonythomas01) [inline vms/volunteer/services.py]: why is there a print ?

**Comment by anjali-dhanuka** [comment]
@kamsuri @abhishekarya286 @tonythomas01 @ayushgarg1804 Please review

**Pull request closed without merging**